ts/packages/cli/src/commands/find.ts

```diff
@@ -0,0 +1,175 @@
+import { Args, Flags } from '@oclif/core';
+import path from 'path';
+import { ConfigSchema } from '@configu/node';
+import { createInterface } from 'readline';
+import { createReadStream } from 'fs';
+import { once } from 'events';
+import _ from 'lodash';
+import { TMPL } from '@configu/ts';
+import FastGlob = require('fast-glob');
+import { BaseCommand } from '../base';
+
+export default class Find extends BaseCommand<typeof Find> {
+  private ignoredFiles: Set<string> = new Set(['build', 'dist', 'node_modules']);
+
+  static description = `Finding all config parameters and their usage in the source code based on CFGU files`;
+
+  static examples = [
+    {
+      description: `Find all config parameters and their usage in current directory`,
+      command: `<%= config.bin %> <%= command.id %>`,
+    },
+    {
+      description: `Find all config parameters and their usage in specific directory from provided CFGU file`,
+      command: `<%= config.bin %> <%= command.id %> <PATH_TO_SCAN> --cfgu <PATH_TO_CFGU>`,
+    },
+    {
+      description: `Find all unused config parameters in specific directory`,
+      command: `<%= config.bin %> <%= command.id %> <PATH_TO_SCAN> --unused`,
+    },
+  ];
+
+  static args = {
+    path: Args.directory({
```

Comment:
Currently we dont use args in the entire CLI and I wish to keep that pattern as args are not named.
Please migrate it to a `dir` flag as in the init command
![image](https://github.com/configu/configu/assets/11806915/8cc25e88-ca23-4529-bf06-8cf8edc8e1b9)

Reply:
sure